drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb

```diff
@@ -164,7 +167,11 @@ def build_assessments(enrollment_scope:, data_collection_stages:, unique_by_info
         # Build FormProcessor with IDs to all related records
         assessment.build_form_processor(**value)
 
-        if assessment.exit? && value[:exit_id].nil?
+        if !assessment.valid?
+          # This check was added because we hit a "Client is invalid", which may have occurred if the client was deleted while the batch was processing?
+          Rails.logger.info "Skipping invalid assessment for EnrollmentID: #{assessment.enrollment_id}"
+          skipped_invalid_assessments += 1
```

Comment:
@eanders Added this in latest commit. I'm seeing validation errors, so just added a check and count.